worker.py

```diff
@@ -79,12 +79,15 @@ def callback(self, task: Task, doc: Document) -> CallbackResponse:
             logger.info(
                 "applying IO on output went well, now finally saving to DANE index"
             )
-            self.save_to_dane_index(
-                doc,
-                task,
-                get_s3_output_file_uri(get_source_id(input_file_path)),
-                provenance=full_provenance_chain,
-            )
+            try:
+                self.save_to_dane_index(
+                    doc,
+                    task,
+                    get_s3_output_file_uri(get_source_id(input_file_path)),
+                    provenance=full_provenance_chain,
+                )
+            except Exception:
+                logger.exception("Failed to save to DANE index")
```

Comment:
When catching a general Exception, at least report back the nature of the exception?

Reply:
this was definitely a quick "fix". It takes some scrutiny & tests to figure out which exceptions can really occur.

Reply:
I've added the Exception's content to the log message, so that it will be slightly easier to understand what goes wrong if this happens